modules/rhpam-apb/roles/deploy-kieserver/tasks/routes.yml

```diff
@@ -6,7 +6,7 @@
       apiVersion: v1
       kind: Route
       metadata:
-        name: '{{ kieserver_deployment_name }}'
+        name: 'secure-{{ kieserver_deployment_name }}'
```

Comment:
Why do you need to prefix it as `secure-` ? Is that related to the scripts?

Reply:
It is to be compatible with existing script. (https://github.com/jboss-container-images/jboss-kie-modules/blob/545387ac2073862138c6e46180d7dfe96bbeaf1f/jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh#L388)

It has been elaborated here.
https://issues.jboss.org/browse/KIECLOUD-97?focusedCommentId=13741158&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13741158